stan/math/prim/scal/fun/rising_factorial.hpp

```diff
@@ -1,13 +1,28 @@
 #ifndef STAN_MATH_PRIM_SCAL_FUN_RISING_FACTORIAL_HPP
 #define STAN_MATH_PRIM_SCAL_FUN_RISING_FACTORIAL_HPP
 
-#include <stan/math/prim/scal/fun/log_rising_factorial.hpp>
-#include <cmath>
+#include <boost/math/special_functions/factorials.hpp>
+#include <stan/math/prim/scal/fun/boost_policy.hpp>
+#include <stan/math/prim/scal/err/check_not_nan.hpp>
+#include <stan/math/prim/scal/err/check_nonnegative.hpp>
+#include <limits>
+#include <string>
 
 namespace stan {
   namespace math {
 
     /**
+     * Return the rising factorial function evaluated 
+     * at the inputs. 
+     * Will throw for NaN x and for negative n
+     *
+     * @tparam T Type of x argument.
+     * @param x Argument.
+     * @param n Argument
+     * @return Result of rising factorial function.
+     */
+
```

Comment:
You need to run the doxygen doc together to make sure the doc goes with the function.  It shouldn't end and then start again with a new `/**`.